src/app/header/header.component.ts

```diff
@@ -27,14 +27,46 @@ export class HeaderComponent implements OnInit {
   urlFeatureToggle: string = '';
   selectedFeatureToggle: string = 'Production';
 
+
+  /*
+   public tabs: Array<any> = [
+   {title: 'Home', content: 'Home content 1', active: true},
+   {title: 'Work', content: 'Work content 2'},
+   {title: 'Code', content: 'Code content 2'},
+   {title: 'Test', content: 'Test content 3'},
+   {title: `Environments/Pipelines`, content: 'Environments/Pipelines content 4'},
+   {title: 'Hypothesis Engine <sup><span class="fa fa-trademark"></span></sup>', content: 'hypo content 4'}
+   ];
+
+   public alertMe(): void {
+   setTimeout(function (): void {
+   alert('You\'ve selected the alert tab!');
+   });
+   };
+
+   public setActiveTab(index: number): void {
+   this.tabs[index].active = true;
+   };
+
+   public removeTabHandler(/!*tab:any*!/): void {
+   console.log('Remove Tab handler');
+   };
+   */
```

Comment:
Did you intend to add this commented code back in or is it an artifact of the merge.